core/bootstrap/src/main/java/org/wildfly/swarm/bootstrap/util/TempFileManager.java

```diff
@@ -38,6 +39,8 @@
 
     public static final TempFileManager INSTANCE = new TempFileManager();
 
+    private static final Pattern tempFilePattern = Pattern.compile("wfswarm\\S+[0-9]{5,}.\\S{5,}");
```

Comment:
I'm not 100% sure about this. Few lines above, we set `WFSWARM_TMP_PREFIX = "thorntail"`, and when I was debugging stuff today, I also noticed the temp directories are named `thorntail*`.

Reply:
I copied it from https://github.com/thorntail/thorntail/blob/master/core/bootstrap/src/main/java/org/jboss/modules/maven/MavenArtifactUtil.java#L65 :-).